lib/utilities/lint-fix.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const chalk = require('chalk');
+const execa = require('execa');
+
+async function run() {
+  let lintFixScriptName = `lint:fix`;
+  let cwd = process.cwd();
+  let npmRunResult = await execa('npm', ['run'], { cwd });
+  let lintFixScriptRegex = new RegExp(`^  ${lintFixScriptName}$`, 'm');
+  let hasLintFixScript = lintFixScriptRegex.test(npmRunResult.stdout);
+
+  if (!hasLintFixScript) {
+    let warningMessage = `'${lintFixScriptName}' not found in package.json scripts`;
+
+    console.warn(chalk.yellow('Lint fix skipped:', warningMessage));
+
+    return Promise.reject({
+      shortMessage: warningMessage,
+    });
```

Comment:
Inside an async context (like here where you are already within an async function execution) you shouldn't return a rejected promise you should throw directly.